highfidelity/hifi
# Case 20242: FST materialMapping supports our material JSON format

https://highfidelity.manuscript.com/f/cases/20242/Add-support-to-the-FST-Serializer-for-our-material-JSON-description-and-targeting

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/6f574a9a41ed930584c699581a24da82/raw/0b3a980a2460394928b8ac0e5dbdd8f3e60653b2/FSTMaterialMapping.js).  You'll see this:
![image](https://user-images.githubusercontent.com/7650116/52664736-a8abb500-2ebe-11e9-87dd-92bff54ac9b6.png)
- Open luci and select the "Scattering" debug framebuffer.  Use your middle mouse button to expand the debug area:
![image](https://user-images.githubusercontent.com/7650116/52664776-c2e59300-2ebe-11e9-90e8-56214b8b248f.png)
- Wear the "Matthew" avatar and look at yourself in mirror mode with the Scattering debug framebuffer open.  You'll see this:
![image](https://user-images.githubusercontent.com/7650116/52664828-de509e00-2ebe-11e9-992d-35c8a73e52d5.png)
